src/commands/DebuggerCommands.ts

```diff
@@ -27,7 +27,7 @@ export namespace DebuggerCommands {
     const web3 = new Web3Wrapper(debugNetworkOptions);
     const providerUrl = web3.getProviderUrl();
 
-    const workspaceFolder = workspace.getWorkspaceFolder(workspaceUri);
+    // const workspaceFolder = workspace.getWorkspaceFolder(workspaceUri);
 
     if (debugNetwork.isLocalNetwork()) {
```

Comment:
After merging https://github.com/trufflesuite/vscode-ext/pull/170, I believe this guard is always `true`, isn't that the case @xhulz? Just asking in case we want to simplify this method even further.

Reply:
Yeah, I think so

I did not analyse the code, once i've made few modification for my code works. But IMHO, all this class needs revalidation.

Thank you

Reply:
Totally, I guess we can take care of this as part of the upcoming release.